### ISSUE TITLE
withErrors built in function doesn't convert as "errors" variable  

### ISSUE THREAD

**Issue by alpemreelmas** [protonemedia/laravel-splade#297]
- Laravel Version: 9.19
- PHP Version: 8.0.2
- Splade JS Version (npm): 1.2.11
- Splade PHP Version (composer): 1.2.11.1

### Description:

I shared piece of my login page code down:

```blade
<x-splade-form action="/login" class="forms-sample">
    <x-splade-errors>
        <div class="bg-red-500 rounded p-3" v-if="Object.keys(errors.all).length > 0">
            <div v-for="(errors, key) in errors.all">
                <p v-for="error in errors" v-text="error" class="text-white"></p>
            </div>
        </div>
    </x-splade-errors>
    <div class="form-group mt-2">
        <label for="exampleInputEmail1">Email address</label>
        <input v-model="form.email" type="text"  class="form-control" id="exampleInputEmail1" placeholder="Email" />
```

Here I can list all my validation errors which works properly but the situation comes to the my special error which I've send via withErrors() built-in function in laravel. I can't see any error in splade-errors. Also I've checked flash variable and there is errors properties which is null. I have no idea that I'm doing something wrong or docs have insufficient information.
Also I tried that when I use with("fail","Some messages"). This works in flash messages.

Thanks for checking and giving feedback. If I have mistake I'm sorry for making you busy.
Regards

**Comment by alpemreelmas** [comment]
> Do you call `withErrors` on a redirect or on a view instance?

I'm using redirect() method.

**Comment by pascalbaljet** [comment]
Thanks for sharing, I'll look into it!

**Comment by pascalbaljet** [comment]
Fixed in v1.2.13, make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).
